src/cdsetool/download.py

```diff
@@ -38,6 +38,7 @@ def download_feature(
     log = _get_logger(options)
     url = _get_feature_url(feature)
     title = feature.get("properties").get("title")
+    temp_dir = _get_temp_dir(options)
```

Comment:
I think `temp_dir = options.get("tmpdir") or None` will give the result you want, and that should also give the nice behavior when the user specifies `"tmpdir": ""` in options.

Reply:
I tried to use the same logic as for log and url that it was created a function specifically but inside the function there is the logic that you mentioned.